staff/marc-uson/spotify-template/style/mixin.sass

```diff
@@ -0,0 +1,23 @@
+@mixin for-galaxy
```

Comment:
@Markuson that is the way! Nice use of mixins!

Reply:
thanks! 🤙